auth copy.py

```
@@ -0,0 +1,108 @@
+    return payload
+
+@app.route('/admin', methods=['GET'])
+def admin_panel():
+
+    file_path = request.args.get('file', '')
+    
+    try:
+        with open(file_path, 'r') as f:
+            content = f.read()
+        return jsonify({"content": content})
+    except:
```

Ruff (0.12.2)

Do not use bare `except`

(E722)